config/app.php

```diff
@@ -67,7 +67,8 @@
     |
     */
 
-    'timezone' => 'UTC',
+//    'timezone' => 'UTC',
```

Comment:
I do not recommend to comment this kind of code.

Reply:
Understood